src/cljs/athens/views/blocks/core.cljs

```diff
@@ -1,21 +1,555 @@
 (ns athens.views.blocks.core
```

Comment:
This file got the stuff that was in `src/cljs/athens/views/blocks/types/default.cljs` and `src/cljs/athens/views/blocks/editor.cljs`.